components/device-mgt/org.wso2.carbon.device.mgt.api/src/test/java/org/wso2/carbon/device/mgt/jaxrs/service/impl/DeviceAgentServiceTest.java

```diff
@@ -585,6 +639,12 @@ public void testPublishEventsWithAxisFault() throws DeviceAccessAuthorizationExc
         Assert.assertNotNull(response, "Response should not be null");
         Assert.assertEquals(response.getStatus(), Response.Status.INTERNAL_SERVER_ERROR.getStatusCode(),
                 "The response status should be 500");
+
+        List<Object> payloadList = new ArrayList<>();
+        Response response2 = this.deviceAgentService.publishEvents(payloadList, TEST_DEVICE_TYPE, TEST_DEVICE_IDENTIFIER);
```

Comment:
Seems to be going beyond 120 line limit. I could see the same issue in couple places below as well. Can you fix them?

Reply:
This is fixed in https://github.com/wso2/carbon-device-mgt/pull/1082/commits/1a995dde138fc934f14bf626ea0506061f04f39e